thrust/thrust/system/cuda/execution_policy.h

```diff
@@ -37,3 +37,67 @@
 #endif // no system header
 #include <thrust/system/cuda/detail/execution_policy.h>
 #include <thrust/system/cuda/detail/par.h>
+
+// define these entities here for the purpose of Doxygenating them
+// they are actually defined elsewhere
+#if 0
```

Comment:
We want this to be guarded by `_CCCL_DOXYGEN_INVOKED`, see #5455
```suggestion
#if _CCCL_DOXYGEN_INVOKED
```